homeassistant/components/hvv_departures/config_flow.py

```diff
@@ -0,0 +1,218 @@
+"""Config flow for HVV integration."""
+import logging
+
+from pygti.auth import GTI_DEFAULT_HOST
+from pygti.exceptions import CannotConnect, InvalidAuth
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import callback
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN  # pylint:disable=unused-import
+from .hub import GTIHub
+
+_LOGGER = logging.getLogger(__name__)
+
+SCHEMA_STEP_USER = vol.Schema(
+    {
+        vol.Required(CONF_HOST, default=GTI_DEFAULT_HOST): str,
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+    }
+)
+
+SCHEMA_STEP_STATION = vol.Schema({vol.Required("station"): str})
+
+SCHEMA_STEP_OPTIONS = vol.Schema(
+    {
+        vol.Required("filter"): vol.In([]),
+        vol.Required("offset", default=0): vol.All(int, vol.Range(min=0)),
```

Comment:
Please import and use `CONF_OFFSET` from const.py:
```suggestion
        vol.Required(CONF_OFFSET, default=0): vol.All(int, vol.Range(min=0)),
```